velox/dwio/parquet/tests/ParquetTpchTestBase.h

```diff
@@ -73,28 +75,65 @@ class ParquetTpchTestBase : public testing::Test {
     auto task = assertQuery(tpchPlan, duckDbSql, sortingKeys);
   }
 
-  void TearDown() {
+  static void TearDownTestSuite() {
     connector::unregisterConnector(kHiveConnectorId);
-    unregisterParquetReaderFactory();
+    velox::parquet::unregisterParquetReaderFactory();
   }
 
  private:
   /// Write TPC-H tables as a Parquet file to temp directory in hive-style
   /// partition
-  void saveTpchTablesAsParquet() {
-    constexpr int kRowGroupSize = 10'000;
-    const auto tableNames = tpchBuilder_.getTableNames();
-    for (const auto& tableName : tableNames) {
+  static void saveTpchTablesAsParquet(memory::MemoryPool* pool) {
+    static constexpr auto tables = {
+        tpch::Table::TBL_PART,
+        tpch::Table::TBL_SUPPLIER,
+        tpch::Table::TBL_PARTSUPP,
+        tpch::Table::TBL_CUSTOMER,
+        tpch::Table::TBL_ORDERS,
+        tpch::Table::TBL_LINEITEM,
+        tpch::Table::TBL_NATION,
+        tpch::Table::TBL_REGION};
+
+    for (const auto& table : tables) {
+      auto tableName = std::string(toTableName(table));
       auto tableDirectory =
           fmt::format("{}/{}", tempDirectory_->path, tableName);
-      fs::create_directory(tableDirectory);
-      auto filePath = fmt::format("{}/file.parquet", tableDirectory);
+
+      auto tableSchema = velox::tpch::getTableSchema(table);
+      auto tt = kDuckDbParquetWriteSQL_.at(tableName);
       auto query = fmt::format(
-          fmt::runtime(kDuckDbParquetWriteSQL_.at(tableName)),
-          tableName,
-          filePath,
-          kRowGroupSize);
-      duckDb_->execute(query);
+          fmt::runtime(kDuckDbParquetWriteSQL_.at(tableName)), tableName);
+      // TODO: The name of API `executeOrdered` is confused, it do nothing
+      // related to sorting, and should be updated with a more proper name.
+      auto result = duckDb_->executeOrdered(query, tableSchema);
+      auto ptr = velox::test::VectorMaker::rowVector(tableSchema, result, pool);
+      result.size();
```

Comment:
dummy statement?

Reply:
`DuckDbQueryRunner->executeOrdered` just executes the query and materializes the result, it does not do any sort. But the API
`DuckDbQueryRunner->execute` sort the result using `multiset`. I want to update the API name of `DuckDbQueryRunner->executeOrdered`, maybe remove the *ordered*. So I added a todo comment, should I remove it?

Reply:
@majetideepak Sorry, I saw the wrong place, I will remove the `result.size()` and update the var names.